students/MichaelUduhiri/Session02/series.py

```diff
@@ -0,0 +1,65 @@
+"""
+a template for the series assignment
+"""
+
+
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    x,y = 1,1
+    for num in range(n-1):
+        x,y = y,x+y
+    return x
+    print (fibonacci(0))
+    pass
```

Comment:
pass not needed here

Reply:
Note fibanacci(0) should return zero.

Reply:
and finish this...

Reply:
Alright. Will take another look.

Reply:
I may have overwritten what I had for grid printer during a previous save request. Will try again